feconf.py

```diff
@@ -693,10 +693,12 @@ def get_empty_ratings():
 DEFAULT_TOPIC_SIMILARITY = 0.5
 SAME_TOPIC_SIMILARITY = 1.0
 
+# While adding another language, please ensure that the
+# languages are in alphabetical order
```

Comment:
Also, this comment doesn't really make sense unless this is a list, because objects in js don't preserve order.